.github/dependabot.yml

```diff
@@ -91,33 +76,18 @@ updates:
           - "*"
 
   - package-ecosystem: "docker"
-    directory: "/sdk/python/runtime"
+    directories:
+      - "/sdk/python/runtime"
+      - "/modules/ruff/build"
     schedule:
       interval: "weekly"
       day: "monday"
       time: "07:00"
       timezone: "UTC"
     labels:
       - "kind/dependencies"
-      - "area/sdk/python"
```

Comment:
Is this intentional?

Reply:
Ah, it was, but I've changed it back, I think it makes sense to leave.

The idea *was* that if we added more non-python docker images, we wouldn't want that label - but ideally in that case, we would just have different PRs for those, with their own labels.